netbox/dcim/filters.py

```diff
@@ -128,6 +128,12 @@ class RackFilter(TenancyFilterSet, CustomFieldFilterSet):
         queryset=Site.objects.all(),
         label='Site (ID)',
     )
+    rack = django_filters.ModelMultipleChoiceFilter(
```

Comment:
This doesn't make sense. The filterset is already being applied to the Rack model; the filter would just match on name. Support for multiple values was introduced in #3038 and will be available in v2.6.0.

Reply:
UI on v2.6-beta1 (d2c66d0baf91) doesn't seem to allow arbitrary rack selection. Changes from #3038 still need to surface to interface somehow.

I'm all ears for better way of achieving this.